### PR TITLE
Create desktop version - Responsive Web Design

### PR DESCRIPTION
**Pull request by clintonjosephs**
- Use media queries to query device screen size
- Add CSS for desktop screen sizes
- Maintain and modify CSS for mobile users
- Use Grid and Flex for layout in desktop view
- Ensure Linters error-free code

### CODE REVIEW

File: assets/css/style.css [hunk 1]
```diff
@@ -4,228 +4,12 @@
...
-  padding: 4px 12px 4px;
-  color: #6070ff;
-}
+/** Shared Button Effects **/
 
 .btn-see-project {
   border: 1px solid #396df2;
   padding: 7px;
-  border-radius: 4px;
+  border-radius: 10px;
```
Comment (VanessaAoki) [inline assets/css/style.css]: - [x] You did an amazing job implementing all the buttons and their actions 👏  but in order to match the Figma design better, you should consider using the style provided there. Did you know that there is a section for buttons only? You can check it out here:

image

File: index.html [hunk 2]
```diff
@@ -4,7 +4,9 @@
         <meta charset="utf-8"> 
         <meta name="viewport" content="width=device-width, initial-scale=1">
         <title>Clinton Mbonu</title>
-        <link rel="stylesheet" href="assets/css/style.css">
+        <link rel="stylesheet" href="assets/css/style.css" />
+        <link rel="stylesheet" href="assets/css/mobile.css" media="screen and (max-width: 992px) ">
+        <link rel="stylesheet" href="assets/css/desktop.css" media="screen and (min-width: 992px) ">
```
Comment (VanessaAoki) [inline index.html:9]: - [x] This is such a smart implementation of media queries, keep up the amazing work 🔥

File: assets/css/desktop.css [hunk 3]
```diff
@@ -0,0 +1,464 @@
...
+  border-radius: 10px;
+  list-style: none;
+  align-items: center;
+  font-size: medium;
+  font-weight: 500;
+  width: 122px;
+  height: 120px;
+  display: inline-block;
+  margin-left: 10px;
+}
```
Comment (VanessaAoki) [inline assets/css/desktop.css]: - [x] Your design looks almost like the original one, to make it perfect, you should consider adding a `padding-top` as well as the right `font-size`.

Figma design:
image

Your design:
image

Current typography for your element:
image

File: assets/css/desktop.css [hunk 4]
```diff
@@ -0,0 +1,464 @@
...
+
+.dflex-item {
+  padding-left: 10px;
```
Comment (VanessaAoki) [inline assets/css/desktop.css]: - [x] As stated in the previous commented, to make it pixel perfect, you should consider adding 2 more pixels here.

File: assets/css/desktop.css [hunk 5]
```diff
@@ -0,0 +1,464 @@
...
+
+/*** Works section **/
+
+.works-flex {
+  background: white;
+  margin: 142px;
+  border-radius: 8px;
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+}
```
Comment (VanessaAoki) [inline assets/css/desktop.css]: - [x] To match the original design better, you should consider adding the `border` provided in the design, as you can see here:

image

File: assets/css/desktop.css [hunk 6]
```diff
@@ -0,0 +1,464 @@
...
+  width: 100%;
+}
+
+.titles-item {
+  font-family: 'Poppins', sans-serif;
+  font-size: 16px;
+  color: lightslategray;
+  font-weight: 500;
+  list-style-image: url("../images/Counter.png");
+  flex: 2;
```
Comment (VanessaAoki) [inline assets/css/desktop.css]: - [x] I believe that this is causing your items to align weirdly. You should consider other alternatives, in order to match the original design better, which has all elements aligned to the left.

Figma design:
image

Your design:
image

File: assets/css/desktop.css [hunk 7]
```diff
@@ -0,0 +1,464 @@
...
+  border-radius: 10px;
+  margin: 24px;
+}
+
+/* Text */
+
+.intro-text-works {
+  font-weight: 650;
+  padding-top: 10px;
+  font-size: 18px;
```
Comment (VanessaAoki) [inline assets/css/desktop.css]: - [x] As mentioned before, you should use the same typography provided by the Figma design, right now your heading is a bit too small.

**Review by VanessaAoki — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED :yellow_circle: 
Hi @clintonjosephs, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

### Project highlights:
- [x] All linter checks are passing :heavy_check_mark: 
- [x] Your README file looks very professional and the PR has a descriptive title/summary of the implemented changes :memo: 
- [x] Descriptive commits and correct use of the GitHub Flow :fire: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

#### You can also consider:
- [x] I can see you added that your repo is MIT licensed, but there is no license file, you should consider adding one. GitHub even provides some license templates, you can read more about it [here](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository).
- [x] You should consider adding more clear instructions on how to start your repo, remember, some of your users might not be familiar with GitHub, so consider adding what exactly should be done in order to clone this repo locally.

so close

Cheers and Happy coding! 👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by clintonjosephs** [comment]
Thank you so much for your feedback, I have made all changes as recommended 👍

**Review by rustam-tolipov — APPROVED** [review]
## Hi @clintonjosephs   👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 🎉 🎉 🎉 🎉 

Cheers and Happy coding!👏👏👏

image

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**